extensions/sql-database-projects/src/tools/netcoreTool.ts

```diff
@@ -24,7 +24,7 @@ export const NetCoreNonWindowsDefaultPath = '/usr/local/share';
 export const winPlatform: string = 'win32';
 export const macPlatform: string = 'darwin';
 export const linuxPlatform: string = 'linux';
-export const minSupportedNetCoreVersionForBuild: string = '3.1.0';
+export const minSupportedNetCoreVersionForBuild: string = '3.1.0'; // TODO: watch out for EOL support in Dec 2022
```

Comment:
are we tracking this elsewhere as well?

Reply:
added link to issue https://github.com/microsoft/azuredatastudio/issues/17800